articles/connections/apple-siwa/add-siwa-to-native-app.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Add Sign In with Apple to Native iOS Apps
+description: Learn how to add native login functionality to your native app with Apple. 
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+# Add Sign In with Apple to Native iOS Apps
+
+You can add functionality to your native iOS application that allows your users to use Sign In with Apple to authenticate. In addition, you can also try the Auth0 [iOS Swift - Sign In with Apple Quickstart](/quickstart/native/ios-swift-siwa).
```

Comment:
Link does not work. Is it something that will be there by the time of merging?

Reply:
Yes